webapp/src/app.tsx

```diff
@@ -1755,10 +1767,30 @@ ${compileService && compileService.githubCorePackage && compileService.gittag ?
 
         return (
             <div id='root' className={rootClasses}>
-                {showExperimentalBanner ? <container.ExperimentalBanner parent={this} /> : undefined}
+                {showExperimentalBanner ? <div id="experimentalBanner" className="ui icon top attached fixed negative mini message">
+                    <sui.Icon icon="warning circle" />
+                    <sui.Icon icon="close" onClick={() => this.hideBanner()} />
+                    <div className="content">
+                        <div className="header">{lf("You are viewing an experimental version of the editor")}</div>
+                        <a href={liveUrl}>{lf("Take me back")}</a>
+                    </div>
+                </div> : undefined}
                 {hideMenuBar ? undefined :
                     <header className="menubar" role="banner">
                         {inEditor ? <accessibility.EditorAccessibilityMenu parent={this} highContrast={this.state.highContrast}/> : undefined }
+
+                        {showWindowsStoreBanner ? <div id="windowsStoreBanner" className="ui attached message">
+                            <sui.Link class="link" target="_blank" ariaLabel={lf("View app in the Windows Store")} href={pxt.appTarget.appTheme.windowsStoreLink}>
+                                <span>
+                                    <img src="https://assets.windowsphone.com/13484911-a6ab-4170-8b7e-795c1e8b4165/English_get_L_InvariantCulture_Default.png"
+                                    alt={lf("Windows Store logo")} />
+                                </span>
+                                {lf("Get the app from the")} <span className="bold">{lf("Windows Store")}</span>
```

Comment:
Composing strings does not translate well for languages that have different word ordering. It has to be a single strings in a lf. For simplicity drop the bold.